renderdoc/driver/shaders/spirv/spirv_disassemble.cpp

```diff
@@ -4659,6 +4659,10 @@ void SPVModule::MakeReflection(GraphicsAPI sourceAPI, ShaderStage stage, const s
         {
           patchData.outTopo = Topology::TriangleStrip;
         }
+        else if(mode.mode == spv::ExecutionModeQuads)
+        {
+          patchData.outTopo = Topology::PatchList;
```

Comment:
I'm on the fence about this - `PatchList` is wrong but I see that lacking an actual `Quads` enum means we need to pick something else. I'm leaning towards saying that this should instead be `TriangleList`. The quad topology is only local within the fixed function tessellator to control the topology of generated primitives correct? so the output is either lines or triangles no matter what.

I don't think there's any code that needs to know what the internal tessellation topology is, and if so this `patchData.outTopo` isn't the place to store it I don't think.

Reply:
Actually, `TriangleList` is correct given that the meaning of `outTopo` is actually the output from the tessellator to the rasterizer.  The "quad" topology only exists between the TCS and TES stages.

So, this would have worked in release builds, despite an error being logged, due to the deliberate fallthrough in `vk_postvs.cpp`.

Reply:
Right that's what I was thinking, the quad is an internal behaviour that determines how the triangles are stitched together. Having this fix is good to avoid relying on the fallthrough as you say, and in future perhaps we might care about the internal representation (one day it'd be nice to debug between TCS and TES).